deploy/artifacts/proxy.yaml

```diff
@@ -0,0 +1,136 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: crane-proxy
+  namespace: openshift-migration
+data:
+  clusters: '[]'
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: proxy
+  namespace: openshift-migration
```

Comment:
This is what you were saying you would need to be able to get the proxy manifests without the rbac stuff included?